### PR TITLE
Adding instructions

### PR DESCRIPTION
**Pull request by lucie-pl**
Instructions added with a new model.
Instructions are now nested attributes of recipe. We can add them on the fly when creating or updating a recipe.
They are rendered on the recipe show page.

Also, unit test and integration test are implemented for instructions.

### CODE REVIEW

File: test/models/instruction_test.rb
Comment (quetzai) [inline test/models/instruction_test.rb:1]: should we keep this file?

**Review by quetzai — APPROVED** [review]
1. We could have split this PR to separate the changes on fixing the ingredient_attributes and the changes on adding instructions alone.
3. I've added one comment to remove a test file not used.

I still approved so you can merge as is or apply the modifications.

File: test/models/instruction_test.rb
Comment (lucie-pl) [reply test/models/instruction_test.rb:1]: It's the instruction unit test, so I think yes. Why not keep it ?

**Review by lucie-pl — COMMENTED** [review]
All good for me. You can merge.

**Pull request merged**